services/movie.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import QuerySet, Q
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int]=None,
+        actors_ids: list[int]=None
+) -> QuerySet:
+    movie_filters = Q()
+
+    if genres_ids:
+        movie_filters &= Q(genres__in=genres_ids)
+
+    if actors_ids:
+        movie_filters &= Q(actors__in=actors_ids)
+
+    return Movie.objects.filter(movie_filters)
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(pk=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list[int]=None,
+        actors_ids: list[int]=None,
+) -> Movie:
+    movie = Movie.objects.create(
+        title=movie_title,
+        movie_description=movie_description,

```

Comment:
There is a typo in the parameter name 'movie_description'. It should be 'description' to match the field name in the `Movie` model.